lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -399,6 +432,36 @@ def add_network_token_cryptogram_and_eci(post, payment_method)
         post[:payment_method_options][:card][:network_token][:electronic_commerce_indicator] = payment_method.eci if payment_method.eci
       end
 
+      def add_network_token(post, payment_method, options)
```

Comment:
Can we rename this to be `add_digital_wallet` or something else since regular NT doesn't flow through these rails?

Reply:
Yep I can! In the original one it did, I really hope we can clean up this gateway soon